Ontology/DTDLv2/RealEstateCore/Information/Georeference.json

```diff
@@ -5,54 +5,30 @@
     {
       "@type": "Property",
       "description": {
-        "en": "Altitude"
-      },
-      "displayName": {
-        "en": "alt"
-      },
-      "name": "alt",
-      "schema": "float",
-      "writable": true
-    },
-    {
-      "@type": "Property",
-      "description": {
-        "en": "The angle between local coordinate system and a north-south meridian."
+        "en": "The angle between the Y-axis of the local coordinate system and a global north-south meridian."
       },
       "displayName": {
         "en": "bearing"
       },
       "name": "bearing",
-      "schema": "float",
-      "writable": true
-    },
-    {
-      "@type": "Property",
-      "description": {
-        "en": "Latitude"
-      },
-      "displayName": {
-        "en": "lat"
-      },
-      "name": "lat",
-      "schema": "float",
+      "schema": "double",
       "writable": true
     },
     {
       "@type": "Property",
       "description": {
-        "en": "Longitude"
+        "en": "A GeoJSON Point coordinate indicating the placement of the local coordinate system origin point (0,0) in the containing global coordinate system. Units: X and Y coordinates in decimal degrees, altitude (if present) in meters."
       },
       "displayName": {
-        "en": "long"
+        "en": "reference point"
       },
-      "name": "long",
-      "schema": "float",
+      "name": "referencePoint",
```

Comment:
"origin" or "originCoordinate" might be more correct. Georeferencing usually takes at least 3 reference points to create an affine transform (in this case specifically referred to as a [Geotransform](https://gdal.org/tutorials/geotransforms_tut.html).)

Reply:
Makes sense to me, we don't want to overload a term that is already used. Though, for usability reasons, keeping "reference" in there somewhere might help indicate that this is not a "self-contained" datum but rather a reference to a more generic world CRS outside of the building. Would something like "originReference" or "originReferencePoint" work?